common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -798,24 +807,42 @@ def test_search(self):
         search_text = 'banana'
         self.create_teams(self.topic, 5)
         self.browse_teams_page.visit()
-        events = [{
-            'event_type': 'edx.team.searched',
-            'event': {
-                'course_id': self.course_id,
-                'search_text': search_text,
-                'topic_id': self.topic['id'],
-                'number_of_results': 0
-            }
-        }, {
-            'event_type': 'edx.team.page_viewed',
-            'event': {
-                'page_name': 'search-teams',
-                'topic_id': self.topic['id'],
-                'team_id': None
-            }
-        }]
-        with self.assert_events_match_during(self.only_team_events, expected_events=events):
-            search_results_page = self.browse_teams_page.search(search_text)
+        search_results_page = self.browse_teams_page.search(search_text)
+        actual_events = self.wait_for_events(
+            event_filter=self.search_event_filter,
+            number_of_matches=3
+        )
+        self.assert_events_match(
+            [
+                {
+                    'event_type': 'edx.team.searched',
+                    'event': {
+                        'course_id': self.course_id,
```

Comment:
With my latest changes, these `course_id`s should be gone unless I missed a spot. :/.